src/components/Header/HeaderClient.tsx

```diff
@@ -72,22 +72,28 @@ export default function HeaderClient({
         <header className={`${className} fixed z-[9999] w-full`}>
             <ScrollShader />
             <div className="mx-auto mt-8 w-responsive">
-                <div className="flex items-center gap-8 border-4 border-black bg-white px-8 py-4 text-grey">
-                    <LogoTitle titleColor="text-grey" className="grow" />
-                    <Links />
-                    {data.nextStep === 'signup' && (
-                        <Button colour="purple" href="/join">
-                            Continue Signing Up
-                        </Button>
-                    )}
-                    {data.nextStep === 'payment' && (
-                        <Button colour="orange" href="/settings">
-                            Continue to payment
-                        </Button>
-                    )}
-                    {data.isSignedIn ? <UserButton data={data} /> : <SignInJoin />}
+                <div className="flex items-center justify-between gap-8 border-4 border-black bg-white px-8 py-4 text-grey">
+                    <div className="flex flex-shrink items-center">
+                        <LogoTitle titleColor="text-grey" className="grow" />
+                    </div>
+                    <div className="flex items-center md:gap-4">
+                        <Links />
+                        <div className="flex gap-4 md:flex-col md:justify-center lg:flex-row lg:items-center">
+                            {data.nextStep === 'signup' && (
+                                <Button colour="purple" href="/join">
+                                    Continue Signing Up
+                                </Button>
+                            )}
+                            {data.nextStep === 'payment' && (
+                                <Button colour="orange" href="/settings">
+                                    Continue to payment
+                                </Button>
+                            )}
+                            {data.isSignedIn ? <UserButton data={data} /> : <SignInJoin />}
+                        </div>
+                    </div>
                 </div>
-                <div className="relative -right-[0.5rem] -top-[4.75rem] -z-10 h-[5.25rem] w-responsive border-4 border-black bg-white" />
+                <div className="relative -right-[0.5rem] -top-[7.5rem] -z-10 h-[8rem] w-responsive border-4 border-black bg-white lg:-top-[4.75rem] lg:h-[5.25rem]" />
```

Comment:
At around 3/4 of screen width and when the words go to a second line, the bottom rectangle is a bit shorter. Can you please fix this?
<img width="892" alt="image" src="https://github.com/compsci-adl/website/assets/47909638/badeebeb-3487-4038-8d6f-a13d4a22bc2c">